examples/tf_mpi/README.md

```diff
@@ -0,0 +1,33 @@
+# MNIST with TensorFlow using MPI through Horovod
+
+This little example show you how to do some distributed training using TensorFlow and Horovod.
+Since Jean-Zay nodes are connected using MPI and with NCCL, synchronous training is theoretically well scalable.
+
+# Step 1: Download the MNIST dataset
+
+Jean-Zay nodes are not connected to the web for security reasons.
```

Comment:
I don't think that this is exactly true.
Indeed a lot of things are blocked by the firewall, but you still have access to the web: for example you can use `git` or `pip`.

Reply:
I think that, for security reasons (I suppose), there are only some specific servers available from the nodes (e.g, pypi.org, github.com, some instances of gitlab...). 
Another option : raw data from MNIST dataset is already available from the folder `$DSDIR/MNIST/`. I suppose that you can load this dataset from Keras (I'm not familiar with this framework but it might be possible).

Reply:
Yeah I tried a lot of hacks but was unable to make the Keras framework downloading the dataset.